docs/sdks/server-sdks/dotnet/initialization.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Initialization
+sidebar_position: 3
+---
+
+import ApiOptionRef from '@site/src/components/ApiOptionRef';
+
+The Eppo .NET SDK is easy to initialize while offering robust customization options, making it adaptable to various use cases such as custom caching requirements and ultra-low-latency initialization.
+
+## Initialize the SDK
+
+To complete basic initialization, you only need to provide an SDK key. [Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
+
+```csharp
+var eppoClientConfig = new EppoClientConfig("<SDK_KEY>", null);
+var eppoClient = EppoClient.Init(eppoClientConfig);
+```
+
+## Use the SDK instance
+After initialization, you can use the SDK instance to assign a variation to a subject using the `Get*Assignment` methods.
+
+```csharp
+// Get the current user from your application context
+var user = GetCurrentUser();
+
+// Make an assignment
+var variation = eppoClient.GetStringAssignment(
+    "my-feature",
+    user.Id,
+    user.Attributes,
+    "default"
+);
+```
+
+## Advanced Configuration
+Basic initialization is great for most use cases, but the SDK provides options that you can use during initialization to customize the behavior of the SDK.
+
+### Initialization Options
+
+How the SDK fetches, serves, and caches experiment configurations is configurable via the `EppoClientConfig` class:
+
+<ApiOptionRef 
+  name="PollingIntervalInMillis"
+  type="int"
+  defaultValue="30000"
+>
+
+The interval in milliseconds between polling attempts to fetch updated configurations from the Eppo API.
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="BaseUrl"
+  type="string"
+  defaultValue="https://fscdn.eppo.cloud/api"
+>
+
+The base URL for the Eppo API. You typically won't need to change this unless you're using a custom endpoint.
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="AssignmentLogger"
+  type="IAssignmentLogger"
+  defaultValue="null"
+>
+
+A callback that sends each assignment to your data warehouse. Required for experiment analysis.
+</ApiOptionRef>
+
+<ApiOptionRef 
+  name="HttpClient"
+  type="HttpClient"
+  defaultValue="new HttpClient()"
+>
+
+The HTTP client used for making requests to the Eppo API. You can provide a custom client to handle things like timeouts and retries.
+</ApiOptionRef>
+
+### Example Configuration
+
+Here's an example of initializing the SDK with custom configuration:
+
+```csharp
+var config = new EppoClientConfig("<SDK_KEY>", new MyAssignmentLogger())
+{
+    PollingIntervalInMillis = 5000,  // Poll every 5 seconds
+    BaseUrl = "https://my-custom-endpoint.com/api",
+    HttpClient = new HttpClient 
+    { 
+        Timeout = TimeSpan.FromSeconds(10) 
+    }
+};
+
+var eppoClient = EppoClient.Init(config);
+```
```

Comment:
@typotter Since the HttpClient is no longer supported, I will need to modify this code sample. Could you suggest what I should change here?

```csharp
var config = new EppoClientConfig("<SDK_KEY>", new MyAssignmentLogger())
{
    PollingIntervalInMillis = 5000,  // Poll every 5 seconds
    BaseUrl = "https://my-custom-endpoint.com/api",
    HttpClient = new HttpClient 
    { 
        Timeout = TimeSpan.FromSeconds(10) 
    }
};

var eppoClient = EppoClient.Init(config);
```

Reply:
The client config code is super simple in its most basic form

`var eppoClientConfig = new EppoClientConfig('SDK-KEY-FROM-DASHBOARD');`